.github/workflows/review.yml

```diff
@@ -112,3 +113,33 @@ jobs:
         with:
           msg: "Created review app at https://ecf-review-pr-${{ github.event.number }}.london.cloudapps.digital"
           check_for_duplicate_msg: true
+
+      - name: Trigger Smoke Test
+        uses: benc-uk/workflow-dispatch@v1.1
+        with:
+          workflow: Smoke Test
+          token: ${{ secrets.PERSONAL_TOKEN }}
+          inputs: '{"paas_environment": "${{ env.ENVIRONMENT }}", "sha": "${{ github.event.pull_request.head.sha }}"}'
+          ref: ${{ github.event.pull_request.head.ref }}
+
+      - name: Wait for smoke test
+        id: wait_for_smoke_test
+        uses: fountainhead/action-wait-for-check@v1.0.0
+        with:
+          token: ${{ secrets.PERSONAL_TOKEN }}
+          checkName: Smoke Test ${{ env.ENVIRONMENT }} ${{ github.event.pull_request.head.sha }}
+          ref: ${{ github.event.pull_request.head.ref }}
+          timeoutSeconds: 300
+          intervalSeconds: 15
+
+      - name: Exit whole workflow if ${{ env.ENVIRONMENT }} smoke test was not successful
+        if: steps.wait_for_smoke_test.outputs.conclusion != 'success'
+        run: exit 1
```

Comment:
Do we get notified if it was not successful? How do we tell?

Reply:
Well, we get told the review app workflow failed (I get emails about it for my PRs), and it should have a little red cross on github. Do you think we should get anything more from review smoke tests?

Reply:
No, that makes sense. I wasn't sure if it would fail the pipeline.